src/main/java/com/slack/kaldb/server/KaldbQueryService.java

```diff
@@ -15,26 +25,73 @@
 
   public static List<String> servers = new ArrayList<>();
 
-  /**
-   * TODO 1. Cache the stub 2. Run in parallel 3. Integrate with ZK to update list 4. Aggregate the
-   * responses instead of only returning the result from the first server
-   */
-  public KaldbSearch.SearchResult distributedSearch(KaldbSearch.SearchRequest request) {
+  // public so that we can override in tests
+  public static int READ_TIMEOUT_MS = 15000;
+
+  public static final KaldbSearch.SearchResult error =
```

Comment:
Is this being used? It appears the same as the `timeOutError` below.

Reply:
I'll fix it! I had it different while I was prototyping an approach where I could distinguish the two with an error message field but decided against it for now atleast